test/onnx/expect/TestOperators.test_std.expect

```diff
@@ -1,6 +1,6 @@
 ir_version: 6
 producer_name: "pytorch"
-producer_version: "CURRENT_VERSION"
+producer_versionproducer_version: "CURRENT_VERSION"
```

Comment:
this doesn't look right: producer_versionproducer_version -> producer_version

Reply:
Thanks. Yeah this is auto generated. Will fix.